### ISSUE TITLE
DefaultReplicationAgents rule doesn't normalize replication target URIs

### ISSUE THREAD

**Issue by actinium15** [adobe/aem-testing-clients#34]
at [0], `DefaultReplicationAgents` configures the replication agents with supplied `serverUrl + /bin/receive` as transport URI.
However, since the `serverUrl` may have a trailing character, [0] should normalize the replication target URI before updating the replication agent config to avoid double slashes in the target URI.

[0] https://github.com/adobe/aem-testing-clients/blob/cq-65/src/main/java/com/adobe/cq/testing/junit/rules/DefaultReplicationAgents.java#L103

**Comment by mohiaror** [comment]
@dulvac, @actinium15  We are still facing this issue in assets' nightly job when the requests to publish are intermittently failing when the URL contains double slash. Can this issue be prioritized?

**Comment by actinium15** [comment]
>We are still facing this issue in nightly job when the requests to publish are intermittently failing when the URL contains double slash.

Hi @mohiaror, I wouldn't expect this issue to appear intermittently if the URLs contain double slashes - it should appear consistently for all urls with double slashes. Can you please reconfirm the statu?
Also, I recall this being fixed within AEM since the time this particular github issue was opened. Can you please confirm if AEM is able to process requests with double slashes or not (perhaps using `curl`)?

**Comment by mohiaror** [comment]
@actinium15 

> I wouldn't expect this issue to appear intermittently if the URLs contain double slashes - it should appear consistently for all urls with double slashes. Can you please reconfirm the statu?

Yes, I can see double slashes even in the logs of the jobs which are successful.

> Also, I recall this being fixed within AEM since the time this particular github issue was opened. Can you please confirm if AEM is able to process requests with double slashes or not (perhaps using curl)

It seems this is where the intermittent behavior is observed. Sometimes the requests are correctly processed and sometimes they throw error. I will investigate more and log an AEM issue for this.

But would you guys suggest that this issue should also be fixed in the meanwhile?
